qa/tasks/deepsea_deploy.py

```diff
@@ -41,6 +41,11 @@
    /srv/salt/ceph/stage/prep/minion/default.sls
 """
 global_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/global.conf'
+mon_conf = '/srv/salt/ceph/configuration/files/ceph.conf.d/mon.conf'
```

Comment:
This is a duplicate. The global variable initializations are in alphabetical order.

If you want to have them all together, you could rename them e.g. `ceph_conf_global_conf`, `ceph_conf_mon_conf`, etc. ?

Reply:
Not clear to me, you mean, we don't need different config files other than global.conf? 

In deepsea README I found that each file should be defined for each section